internal/driver/fetch_test.go

```diff
@@ -416,6 +423,10 @@ func TestHttpsInsecure(t *testing.T) {
 	if len(p.SampleType) == 0 {
 		t.Fatalf("fetchProfiles(%s) got empty profile: len(p.SampleType)==0", address)
 	}
+	if runtime.GOOS == "plan9" {
```

Comment:
There is badSigprofOS map below which lists plan9. That map is for environments where the CPU profiling is supported but produces incorrect results. I suggest we remove plan9 from there since it's now handled here.

Reply:
Done.